src/main/java/io/jenkins/pluginhealth/scoring/schedule/ProbeEngineScheduler.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2022 Jenkins Infra
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.jenkins.pluginhealth.scoring.schedule;
+
+import io.jenkins.pluginhealth.scoring.probes.ProbeEngine;
+
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ProbeEngineScheduler {
+    private final ProbeEngine probeEngine;
+
+    public ProbeEngineScheduler(ProbeEngine probeEngine) {
+        this.probeEngine = probeEngine;
+    }
+
+    @Scheduled(cron = "${cron.probe-engine}", zone = "UTC")
+    public void run() {
+        probeEngine.run();
```

Comment:
1. Yes. Some probe will not require a new release of a plugin to be updated (like the last commit date). So we could run the probe engine more frequently than the update of the update-center.
2. `run` because naming things is hard and this name was obvious to me: it runs the probe engine.
    . not at all. The `run` method in `UpdateCenterScheduler` is implementing the method from the `CommandLineRunner` interface. This method is not implementing nor overriding anything. This method will be called because of the `@Scheduled` annotation.

Reply:
1. Oh yes, I forgot about those cases. Makes sense now.
2. That makes me wonder, now that we're running the `updateDatabase` method via @ Scheduled, why I even have the `UpdateCenterScheduler` class implement the `CommandLineRunner` interface and calling the `updateDatabase` method via the overridden `run` method. Discussion for future I think.